src/registryDAO/types.mligo

```diff
@@ -47,6 +47,10 @@ type lookup_registry_param =
 
 type lookup_registry_view = (registry_key * (registry_value option)) contract
 
+type custom_ep_param =
+  | Lookup_registry of lookup_registry_param
+  | RegistryCepDummy of unit
```

Comment:
What is this for? :thinking:

Reply:
It is so that we can annotate the "Lookup_registry" entrypoint. Or is there another way to annotate this entrypoint from this parameter nesting level?

Reply:
Ah, I see, I don't think there are other way to annotate this with Ligo syntax.

Possibly there's some way to do this with some `#if` clauses, but it doesn't seem straightforward, so I think we can avoid this not and perhaps get back to it later.

Please add a comment to explain this tho, I think it should have a TODO too.

Reply:
Have added a comment and a TODO.